components/script/dom/xrsession.rs

```diff
@@ -73,6 +73,9 @@ pub struct XRSession {
     end_promises: DomRefCell<Vec<Rc<Promise>>>,
     /// https://immersive-web.github.io/webxr/#ended
     ended: Cell<bool>,
+    /// Opaque framebuffers need to know the session is "outside of a requestAnimationFrame"
+    /// https://immersive-web.github.io/webxr/#opaque-framebuffer
+    outside_raf: Cell<bool>,
```

Comment:
kinda would prefer this boolean to be inverted, but no biggie

Reply:
I was following the way round it is in the spec, which is just as backwards!

Reply:
works for me